### DISCUSSION TITLE
Floating nodes now gives random values

### DISCUSSION THREAD

**Discussion by mengstr** [hneemann/Digital#698] [General]
I decided to restart the development of my PDP-8 project from 1.5 years ago and tried it on the current version of Digital. It did't go too well. ;-). 

Seems like I had a bunch of wires that sometimes were floating and in the old version this was interpreted as a low, but today it's giving a random low/high to mess up your design if one was careless.  A bit annoying, but I see the reason for it and I give it a solid :+1:

With the new version of Digital I of course lost my own patches to the codebase, but I managed to do without them for the time being.  But I still think that my "Counting Probe" would be a good addition to Digital. And I also like my Telnet terminal peripheral device.  I'll give it a go to convert those into plugins for Digital and see if I can manage that or not.

**Comment by hneemann** [comment]
I was able to find the CntProbe which i have to think about again, but I could not find the Telnet terminal peripheral device.

**Reply by mengstr** [reply]
I've updated the CntProbe to compile correctly with the current main and did a PR for it.

I'll extract and fixup the old Telnet Terminal code and do a PR for it as well for your consideration.

**Reply by hneemann** [reply]
I have already implemented an appropriate CntProbe, and it feels wrong to me. There should be a more general solution that can be used for more flexible debugging. But I'm not sure yet how this could look like.

You don't need to put so much work into the telnet terminal implementation. It's enough for now if you describe what exactly this component does.

**Reply by mengstr** [reply]
I've made a PR for the telnet/socket device and also tried to describe what it does there.  It was not much work and now I'm uptodate with your source which is good for my own purposes. A lot of bitrot happens in code that has not been used for 1.5 years.  This time I'll ty to keep my fork better in sync with your main.
